cclient/iota_client_extended_api.h

```diff
@@ -21,9 +21,9 @@ typedef flex_hash_array_t address_list_t;
 typedef flex_hash_array_t inputs_list_t;
 typedef flex_hash_array_t transfer_list_t;
 typedef flex_hash_array_t transaction_list_t;
-typedef flex_hash_array_t transaction_objs_t;
 typedef flex_hash_array_t booleans_t;
 typedef flex_hash_array_t input_list_t;
+typedef UT_array transaction_objs_t;
```

Comment:
We already have `typedef UT_array bundle_transactions_t;` and related methods in `common/model/bundle.h`. Would it make sense, for semantic, to use them ?

Reply:
`bundle_transactions_t` is an array of `iota_transaction_t`, the exact same thing as `transaction_objs_t`. But since here we are dealing with bundles maybe we should use it ?

Reply:
ah..right, it is a bundle operation. sorry.